cves/CVE-2015-5346.yml

```diff
@@ -124,10 +125,10 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
-  contest:
+  answer: This vulnerability was discovered by the tomcat security team. They discovered when they realized Tomcat incorrectly handled different session settings when multiple versions of the same web application was deployed
```

Comment:
Just need to change second sentence to "They discovered it..." or "It was discovered".